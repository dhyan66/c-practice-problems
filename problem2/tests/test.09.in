100 400
48
19
83
20
8
90
47
51
32
34
34
39
47
47
35
35
82
65
57
69
30
54
96
79
14
73
61
47
51
94
78
38
83
73
4
19
89
45
5
24
16
60
28
21
52
37
85
97
36
78
48
23
72
69
32
60
69
46
18
11
85
93
60
99
75
86
17
82
55
61
0
54
55
77
57
74
41
5
25
22
35
86
21
19
32
87
1
33
83
29
56
15
86
76
20
53
97
83
54
76
45
33
60
51
90
32
79
15
42
66
89
41
44
19
43
53
99
28
73
13
93
66
16
59
80
50
94
44
54
27
72
44
65
9
2
22
95
52
34
30
41
27
4
23
88
18
68
44
97
21
23
73
19
24
42
77
11
83
5
93
64
69
22
35
33
15
3
96
14
60
43
99
92
59
22
77
13
34
43
92
75
49
24
7
30
40
16
14
63
4
62
69
25
79
90
10
56
24
37
98
99
4
15
60
0
44
11
47
52
71
66
15
77
5
79
77
66
86
34
60
50
2
41
71
34
81
15
58
5
35
98
33
28
23
19
68
92
4
31
94
65
64
91
43
6
54
7
96
44
8
10
84
84
91
21
48
61
31
62
84
36
59
25
9
99
44
44
93
22
68
85
31
12
80
91
16
9
71
27
80
75
63
74
33
30
54
13
45
53
69
83
55
37
41
68
98
1
83
69
71
39
18
84
48
23
30
97
46
19
70
15
37
4
42
13
61
36
20
96
70
33
83
85
1
5
74
43
26
67
85
8
64
29
7
69
93
7
27
41
40
29
60
51
29
53
62
8
0
24
85
99
87
76
80
54
56
71
65
62
86
74
50
70
43
26
13
13
21
59
31
22
47
56
60
26
23
35
17
58
16
57
85
84
66
22
36
48
87
52
68
2
39
47
61
60
81
39
63
64
62
