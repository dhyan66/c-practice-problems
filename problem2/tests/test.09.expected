Odd: 0 1 2 3 6 9 11 12 14 15 16 18 19 20 21 22 24 25 26 28 30 38 42 45 47 48 49 50 54 55 57 61 62 63 66 67 68 70 71 75 76 77 84 85 86 87 88 90 91 92 93 94 95 98
